frontend/features/sectionBlocks/CardBlock.tsx

```diff
@@ -73,14 +56,16 @@ export function CardBlock({ imageUrl, projectLink }: CardBlockProps) {
   );
 }
 
+function BlockComponent({ index, data }: BlockComponentProps) {
+  return (
+    <BlocksControls index={index} focusRing={{ offset: 0 }} insetControls>
+      <CardBlock {...data} projectLink={data.projectLink} />
```

Comment:
`projectLink={data.projectLink}` is redundant, since we're already spreading `data` as props, so why are we doing it?